libs/application/ui-fields/src/lib/SubmitFormField/SubmitFormField.tsx

```diff
@@ -38,7 +38,8 @@ export const SubmitFormField: FC<React.PropsWithChildren<Props>> = ({
       borderRadius="large"
       width="full"
       padding={4}
-      marginTop={4}
+      marginTop={marginTop ?? 4}
```

Comment:
Again, would it be cleaner to replace this with defaulting in the destructuring?

Reply:
Fixed